### DISCUSSION TITLE
error in s3

### DISCUSSION THREAD

**Discussion by KcVamsi** [learndevopsonline/learndevopsonline#1837] [Q&A]
<img width="557" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/129276861/429cef3b-891a-4490-94d6-f5655a221b40">

Hello @r-devops @chaithanya1812 

Can you please explain why we are getting the s3 bucket error and also why we are using the -backend-config=env-dev/state.tfvars in the cli.

**Comment by chaithanya1812** [comment]
### Hi Vamsi, why don't you join lab session once, 
### so that your doubts will be cleared..
